src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -280,7 +280,11 @@ public ConnectorType ConnectorType
         /// </summary>
         public readonly List<WorkspaceModel> Workspaces =
             new List<WorkspaceModel>();
-        
+
+        public bool RunEnabled
+        {
+            get { return Workspaces.OfType<HomeWorkspaceModel>().FirstOrDefault().RunEnabled; }
+        }
```

Comment:
Now that we have promoted `RunEnabled` to `WorkspaceModel` base class, I was originally expecting this to be written as:

```csharp
public bool RunEnabled
{
    get
    {
        if (currentWorkspace == null)
            return false;

        return currentWorkspace.RunEnabled;
    }
}
```